support-frontend/assets/pages/supporter-plus-thank-you/components/thankYouHeader/heading.tsx

```diff
@@ -199,122 +48,78 @@ type HeadingProps = {
 function Heading({
 	name,
 	productKey,
-	isOneOffPayPal,
 	amount,
 	currency,
 	isObserverPrint,
 	ratePlanKey,
-	paymentStatus,
 	promotion,
 }: HeadingProps): JSX.Element {
-	const isPending = paymentStatus === 'pending';
 	const isDigitalEdition = productKey === 'DigitalSubscription';
 	const isGuardianAdLite = productKey === 'GuardianAdLite';
 	const isTier3 = productKey === 'TierThree';
 
+	const contributionProduct = isContributionProduct(productKey);
 	const printProduct = isPrintProduct(productKey);
-	const maybeNameAndTrailingSpace: string =
-		name && name.length < 10 ? `${name} ` : '';
-	const maybeNameAndCommaSpace: string =
-		name && name.length < 10 ? `, ${name}, ` : '';
 
-	// Print Products Header
+	const contributorName = name && name.length < 10 ? name : '';
+
 	if (printProduct) {
 		return (
 			<h1 css={longHeaderTitleText}>
```

Comment:
Possibly this <`h1`> tag be placed inside `<PrintProductHeading>` component, so looks like same level as `<ContributionHeading>`